### PR TITLE
Pass Client UTC Offset inside join message over signalling channel

### PR DESCRIPTION
**Pull request by aman-aguptaaa**
**Issue #:** None

**Description of changes:**
*What?*
Add Client UTC Offset inside join message (which is sent over signalling channel)

*Why?*
With the aim of improving audio quality for customers, we want to map the timezone of attendees to server's AWS region  being connected to by them. 

**Testing:**
- Added unit tests for all modified code with full coverage
- Used demo application to verify that the client UTC offset parameter is being passed inside join message.

*Can these tested using a demo application? Please provide reproducible step-by-step instructions.*
Yes, Running MeetingV2 Demo App in the browser should pass the client UTC offset parameter inside join message. You can filter the logs in console of browser by searching for the term `client_utc_offset`

**Checklist:**

1. Have you successfully run `npm run build:release` locally?
Yes

2. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
 N/A

3. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
 N/A

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [3.13.0] - 2023-03-28
 
 ### Added
+- Add Client UTC Offset inside join message which is sent over signalling channel.
```
Comment (devalevenkatesh) [inline CHANGELOG.md:11]: Can make this builder friendly: "Send client UTC offset with attendee `JOIN` frame over signalling channel."

File: package.json [hunk 2]
```diff
@@ -102,6 +102,7 @@
     "@aws-sdk/util-hex-encoding": "^3.47.0",
     "@types/ua-parser-js": "^0.7.35",
     "detect-browser": "^5.2.0",
+    "moment-timezone": "^0.5.42",
```
Comment (devalevenkatesh) [inline package.json:105]: Is there a browser native API over adding a dependency? If not hard enough, we would like to limit the dependencies.

File: src/utils/Utils.ts [hunk 3]
```diff
@@ -1,6 +1,8 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
+import moment = require('moment-timezone');
```
Comment (devalevenkatesh) [inline src/utils/Utils.ts:4]: Its 2.99 MB when un-packed dependency per NPM, could we do this on backend?

File: package.json (hunk 2 above)
Comment (aman-aguptaaa) [reply package.json:105]: Removed the dependency.

File: src/utils/Utils.ts (hunk 3 above)
Comment (aman-aguptaaa) [reply src/utils/Utils.ts:4]: Removed the dependency.

File: CHANGELOG.md (hunk 1 above)
Comment (aman-aguptaaa) [reply CHANGELOG.md:11]: Changed. Thanks.

File: src/utils/Utils.ts [hunk 4]
```diff
@@ -36,3 +36,20 @@ export function toLowerCasePropertyNames(input: any): any {
     return result;
   }, {});
 }
+
+/**
+ * Get UTC offset in (+|-)HH:mm format
+ * E.g. For Asia/Calcutta timezone, +05:30 UTC offset value is returned
+ */
+export function getUTCOffsetString(utcOffsetInteger: number): string {
```
Comment (devalevenkatesh) [inline src/utils/Utils.ts:44]: - Can function be named to `getFormattedOffset(utcOffset: number): string`?
- Do we have to consider DST difference?

Comment (aman-aguptaaa) [reply src/utils/Utils.ts:44]: getTimezoneOffset considers daylight savings as well. Ref [this](https://stackoverflow.com/questions/11887934/how-to-check-if-dst-daylight-saving-time-is-in-effect-and-if-so-the-offset).

Comment (aman-aguptaaa) [reply src/utils/Utils.ts:44]: Will rename to getFormattedOffset

**Review by devalevenkatesh — APPROVED** [review]

**Review by ltrung — APPROVED** [review]

**Pull request merged**